assets/app/styles/_spacers.less

```diff
@@ -0,0 +1,36 @@
+// Spacing utility classes
+// creates:
+//  .pad-sm, .pad-md, .pad-lg, etc
+//  .mar-sm, .mar-md, .mar-lg, etc
+//  .pad-left-sm, .mar-left-sm, etc
+//
+@sizes: none 0px e('!important'), sm 4px e(''), md 8px e(''), lg 12px e(''), xl 16px e(''), xxl 32px e('');
```

Comment:
Yeah i definitely wanted to cover the 5,10,15,20... and also be flexible enough for the deviants like 2,3,8,12.  Half steps are always helpful, but `xs,sm,md,lg,xl,xxl` seems like it should be enough for what we need.  Perhaps there is a better naming convention to give us a few more options?